src/main.c

```diff
@@ -1307,6 +1307,9 @@ static gboolean info_start(int argc, char **argv)
 	if (no_check_time)
 		check_bundle_params |= CHECK_BUNDLE_NO_CHECK_TIME;
 
+	g_assert(access_args.http_info_headers == NULL);
+	access_args.http_info_headers = assemble_info_headers(NULL);
```

Comment:
What's the scenario this would be useful for? When using the CLI, the context information will not be ready. When using RAUC without the service, this might not be relevant anymore (since e.g. polling will not work).

Reply:
For `rauc install`, `r_context_configure_target()` is called even without the service, as it uses `R_CONTEXT_CONFIG_MODE_REQUIRED`. For `rauc info` (which uses `_AUTO`), the target information (mainly the system-info handler) is not loaded. So we probably only get the uptime.

We could change that by calling `r_context_configure_target()` in more cases, but that can be left for later.

Reply:
Documenting this clearly might be sufficient for now.